Example/__tests__/Animation.test.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent } from '@testing-library/react-native';
+import { renderHook } from '@testing-library/react-hooks';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  JestReanimated,
+} from 'react-native-reanimated';
+const {
+  withReanimatedTimer,
+  moveAnimationByTime,
+  moveAnimationByFrame,
+  getAnimatedStyle,
+} = JestReanimated;
+
+export default function TestComponent1(props) {
+  const widthSV = props.sharedValue ? props.sharedValue : useSharedValue(0);
+
+  const style = useAnimatedStyle(() => {
+    return {
+      width: withTiming(widthSV.value, { duration: 500 }),
+    };
+  });
+
+  return (
+    <View style={{ flex: 1, flexDirection: 'column' }}>
+      <Animated.View
+        testID="view"
+        style={[
+          { width: 0, height: 80, backgroundColor: 'black', margin: 30 },
+          style,
+        ]}
+      />
+      <Button
+        testID="button"
+        title="toggle"
+        onPress={() => {
+          widthSV.value = 100;
+        }}
+      />
+    </View>
+  );
+}
+
+describe('Tests of animations', () => {
+  test('withTiming animation', async () => {
+    jest.useFakeTimers();
+    const style = {
+      width: 0,
+      height: 80,
+      backgroundColor: 'black',
+      margin: 30,
+    };
```

Comment:
Nit; We frequently use the same style (with `width` change at most), maybe some function returning it would DRY it up?